python/ray/workflow/tests/test_virtual_actor_2.py

```diff
@@ -6,6 +6,11 @@
 from ray import workflow
 
 
+def skip_client_tests():
```

Comment:
Try to use this pattern to skip test: https://github.com/ray-project/ray/blob/master/python/ray/tests/test_actor.py#L672

Reply:
@iycheng I feel in our cases the client mode is "activated" during runtime by the fixtures, so we need a runtime skip. The decorator way I think is evaluated during import/decorator time.  But let me follow the pattern and give it try just to confirm.